demo/runtime/src/lib.rs

```diff
@@ -295,6 +296,44 @@ impl frame_system::Config for Runtime {
 	type PostTransactions = ();
 }
 
+#[dynamic_params(RuntimeParameters, pallet_parameters::Parameters::<Runtime>)]
+pub mod dynamic_params {
+	use super::*;
+
+	#[dynamic_pallet_params]
+	#[codec(index = 0)]
+	pub mod cardano_config {
+		#[codec(index = 0)]
+		pub static SecurityParameter: u64 = 432;
+		#[codec(index = 1)]
+		pub static ActiveSlotsCoeff: sp_runtime::Permill = sp_runtime::Permill::from_percent(5);
+	}
+
+	#[dynamic_pallet_params]
+	#[codec(index = 1)]
+	pub mod cardano_epoch_config {
+		#[codec(index = 0)]
+		pub static EpochDuration: core::time::Duration =
+			core::time::Duration::from_millis(432000000); // 5 days
+		#[codec(index = 1)]
+		pub static SlotDurationMillis: core::time::Duration =
+			core::time::Duration::from_millis(1000); // 1 second
+		#[codec(index = 2)]
+		pub static FirstEpochTimestampMillis: u64 = 1596059091000;
+		#[codec(index = 3)]
+		pub static FirstEpochNumber: u32 = 208;
+		#[codec(index = 4)]
+		pub static FirstSlotNumber: u32 = 4492800;
+	}
+}
+
```

Comment:
For testing purposes I have tried to add a structure, [here](https://github.com/input-output-hk/partner-chains/commit/40a3ea79d02d397d7b15c2d78221aee849fc2a9b#diff-7a669d0e1ac97f6a188dda132485100d776692c570f0417c9e1c284711259ceeR314) I evaluated it to a mod with elements, but in case you will need some aggregated parameter, looks like anything that implements below set of traits should work:
```suggestion
		#[codec(index = 5)]
		pub static FooBBBB: FooBar = Default::default();
	}
}

#[derive(
	Default, Clone, TypeInfo, Encode, Decode, DecodeWithMemTracking, MaxEncodedLen, PartialEq, Eq,
)]
pub struct FooBar(u64);
```